src/test/java/org/mule/extensions/jms/test/connection/JmsConnectionFactoryTestCase.java

```diff
@@ -0,0 +1,36 @@
+/*
+ * Copyright (c) MuleSoft, Inc.  All rights reserved.  http://www.mulesoft.com
+ * The software in this package is published under the terms of the CPAL v1.0
+ * license, a copy of which has been included with this distribution in the
+ * LICENSE.txt file.
+ */
+package org.mule.extensions.jms.test.connection;
+
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.hamcrest.Matchers.is;
+import static org.mule.extensions.jms.test.AllureConstants.JmsFeature.JMS_EXTENSION;
+import static org.mule.extensions.jms.test.AllureConstants.JmsFeature.JmsStory.CONNECTION_FACTORY;
+
+import org.mule.extensions.jms.test.JmsAbstractTestCase;
+import org.mule.extensions.jms.test.infra.JmsTestConnectionFactory;
+
+import io.qameta.allure.Description;
+import io.qameta.allure.Feature;
+import io.qameta.allure.Story;
+import org.junit.Test;
+
+@Feature(JMS_EXTENSION)
+@Story(CONNECTION_FACTORY)
+public class JmsConnectionFactoryTestCase extends JmsAbstractTestCase {
+
+  @Override
+  protected String getConfigFile() {
+    return "config/generic/custom-connection-factory-properties.xml";
+  }
+
+  @Test
+  @Description("verify that connection factory properties are actually passed to the underlying ConnectionFactory.")
+  public void testConnectionFactoryPropertiesPassed() throws Exception {
```

Comment:
As we talked in private, this test doesn't make any sense.
The only thing that you are testing is the Spring Module, to check if it registers correctly the spring bean in the registry.
Change the tests to verify that the `ConnectionFactory` was injected correctly inside the JMS Connection or delete it

Reply:
You're right @estebanwasinger. I'm sorry, I forgot to delete it. 

Actually, I spent some time yesterday trying to figure it out a way to make that assertion, Indeed I got some help from Rodrigo Merino and we realize there's no way to do it unless we change the approach to a pure unitary white box test for the JMS Connector. 

Having that said. I'd prefer to delete it for the scope of this issue.